### PR TITLE
Second draft

### PR DESCRIPTION
**Pull request by smpark7**
This is my second draft for review.

Addresses #3

### CODE REVIEW

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: What does it mean for an abstract to present an *outline* of verification?

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: consider using the glossaries package for auto-expanding acronyms.

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: Sentence is too convoluted. 
```suggestion
MSR simulation tools must capture strong coupling between neutronics and thermal-hydraulics exhibited by delayed neutron precursor movement as well as strong Doppler and density feedbacks in the fuel salt.
```

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: unnecessary passive voice.

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]:
```suggestion
The fuel composition in an MSR continuously evolves over time as online refueling replenishes fissile material while removing fission product poisons. The impact of changing fuel composition on the safety parameters such as the reactivity feedback coefficient must be fully characterized to establish a licensing case for this class of reactors.
```

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: - unnecessary passive voice.
- don't use the word "code" in this way.  (application is a noun... there's no reson to add the word code... and "software package" or "computational tool" or similar is preferable to "code" )

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: unnecessary passive voice.

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: Unnecessary passive voice. These sentences are also very monotonous when stacked back to back in this way.. "Moltres is... Moltres is.... MOOSE is..." Try reading them aloud and you'll see what I mean..

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: This sentence is a bit of a "so what". (connect this sentence to the topic of your abstract.)

File: abstract.tex
Comment (katyhuff) [inline abstract.tex]: Why so much passive voice?
In #2 I expressed that there was too much passive voice in the first draft.
This PR (which was supposed to address #2, not #3 by the way... ) seems to go entirely in the other direction, adding passive voice where active voice used to appear.... What is goign on? I am going to stop reading here. Please fix this before the next request for a review.

**Review by katyhuff — CHANGES_REQUESTED** [review]
Why so much passive voice?
In #2 I expressed that there was too much passive voice in the first draft.
This PR (which was supposed to address #2, not #3 by the way... ) seems to go entirely in the other direction, adding passive voice where active voice used to appear.... What is going on? I stopped reading halfway through because it's clear my previous comment was not addressed. Please fix this before the next request for a review.

**Comment by smpark7** [comment]
I apologize for the passive voice sentences. I was trying to remove them but I failed to recognize them and inadvertently added more.

**Comment by smpark7** [comment]
The deadline has been extended until 15 march.

**Comment by katyhuff** [comment]
I will merge this and submit corrections with a PR. Please do not submit until that (upcoming) PR is merged.

**Pull request merged**
